### PR TITLE
Fix dropdown going off screen

### LINKED ISSUE #132: Autocomplete UI goes off page
https://github.com/thelounge/lounge/issues/1890

### PR DESCRIPTION
**Pull request by xPaw**
Fixes #132

### CODE REVIEW

File: src/dropdown.js [hunk 1]
```diff
@@ -182,9 +182,17 @@ export default class Dropdown extends EventEmitter {
 
   /** @private */
   setOffset(cursorOffset: CursorOffset) {
+    const elementWidth = this.el.offsetWidth;
     if (cursorOffset.left) {
+      const browserWidth = document.documentElement.clientWidth;
```
Comment (yuku-t) [inline src/dropdown.js]: `document.documentElement` is nullable. I think we should merge L187 and L199, for example:

```js
setOffset(cursorOffset: CursorOffset) {
  const doc = document.documentElement;
  if (!doc) { return; } // non browser
  const elementWidth = this.el.offsetWidth;
  if (cursorOffset.left) {
    const browserWidth = doc.clientWidth;
  // ...
  if (this.isPlacementTop()) {
    this.el.style.bottom = `${doc.clientHeight - cursorOffset.top + cursorOffset.lineHeight}px`
  // ...
}
```

Comment (xPaw) [reply src/dropdown.js]: Done

**Comment by MaxLeiter** [comment]
before: 
screen shot 2018-03-23 at 10 42 58 am


after: 
screen shot 2018-03-23 at 10 42 24 am

**Comment by yuku-t** [comment]
@MaxLeiter Thanks!

**Comment by yuku-t** [comment]
v0.17.0 was released

**Pull request merged**